macos/Sources/Ghostty/Ghostty.Config.swift

```diff
@@ -381,7 +381,20 @@ extension Ghostty {
             let backgroundColor = OSColor(backgroundColor)
             let isLightBackground = backgroundColor.isLightColor
             let newColor = isLightBackground ? backgroundColor.darken(by: 0.08) : backgroundColor.darken(by: 0.4)
-            return Color(newColor)
+
```

Comment:
The comments at the beginning of the function should be fixed up to indicate that this is now configurable.